core/include/detray/navigation/intersection/helix_cylinder_intersector.hpp

```diff
@@ -89,7 +84,7 @@ struct helix_intersector_impl<cylindrical2D<algebra_t>, algebra_t>
         // Try to guess the best starting positions for the iteration
 
         // Direction of the track at the helix origin
```

Comment:
Please leave the deleted part intact and trigger the newton-raphson-saft method only for (run_rtstate == true)
And run the original part for (run_rtstate == false)